go/src/client/main.go

```diff
@@ -96,6 +96,8 @@ func uploadGame(httpClient *http.Client, path string, pgn string, nextGame clien
 	body := &bytes.Buffer{}
 	_, err = body.ReadFrom(resp.Body)
 	if err != nil {
+		time.Sleep(time.Millisecond * 2000 << retryCount)
```

Comment:
Precedence always a bit tricky with shifts, maybe `time.Second * (2 << retryCount)`?

Reply:
I don't think order of operations matters here at all. Either way the shift is just a fast multiply by 2^power and multiplication is commutative.